diff-containers/src/Data/Map/Diff/Strict.hs

```diff
@@ -1,3 +1,20 @@
+{-# OPTIONS_GHC -Wno-unused-imports #-}
```

Comment:
Left-over?

Reply:
The imports of `Data.Map.Strict` and `Data.Group` are unused, but they are useful for hyperlinking in the haddock comments, so this option is still necessary